core/app/models/workarea/order/item.rb

```diff
@@ -166,5 +166,22 @@ def customizations_eql?(test)
         test.blank? && customizations.blank?
       end
     end
+
+    # Determine whether the additional fields on this item are
+    # equivalent to the passed-in Hash of parameters. Uses the
+    # `distinct_order_item_attributes` configuration setting to filter
+    # down which parameters it will be checking equivalence on.
+    #
+    # @param [Hash] params - Updated order item parameters.
+    # @return [Boolean] whether this item's attributes is equal to the
+    #                   passed-in parameters.
+    def attributes_eql?(params = {})
```

Comment:
This name is a little unclear, how can we name it represent that this is different from `order_item.attributes == other_order_item.attributes`

Reply:
Agreed, not really sure what to call it though. I used this method name as a stand-in for a future name.

Reply:
Time to dance the side convo shuffle.